doc/codi.txt

```diff
@@ -245,7 +245,14 @@ g:codi#autocmd
 
                                                                 *g:codi#width*
 g:codi#width
-             The width of the Codi split.
+             The width of the Codi split, either a |Number| or a |Float|.
+
+             When a |Float| is supplied, it will be treated as a percentage.
```

Comment:
Let's specify this as "percentage of the current buffer width". Considering the user may already have a vertical split open, it may be alarming if they think you mean percentage of the window.